library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -123,6 +124,8 @@ public static void fadeInAnim(final View view) {
      * @param view view that starts that animation.
      */
     public static void shrinkAnim(final View view, final boolean removeView) {
+        RotateDrawable rotateDrawable = new RotateDrawable();
+        rotateDrawable.setFromDegrees(10);
```

Comment:
**Reporter**: Android Lint
**Rule**: Correctness
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L128

NewApi: Calling new methods on older versions
Call requires API level 21 (current min is 15): android.graphics.drawable.RotateDrawable#setFromDegrees
This check scans through all the Android API calls in the application and warns about any calls that are not available on **all** versions targeted by this application (according to its minimum SDK attribute in the manifest).&#xA;&#xA;If you really want to use this API and don't need to support older devices just set the `minSdkVersion` in your `build.gradle` or `AndroidManifest.xml` files.&#xA;&#xA;If your code is **deliberately** accessing newer APIs, and you have ensured (e.g. with conditional execution) that this code will only ever be called on a supported platform, then you can annotate your class or method with the `@TargetApi` annotation specifying the local minimum SDK to apply, such as `@TargetApi(11)`, such that this check considers 11 rather than your manifest file's minimum SDK as the required API level.&#xA;&#xA;If you are deliberately setting `android:` attributes in style definitions, make sure you place this in a `values-v`*NN* folder in order to avoid running into runtime conflicts on certain devices where manufacturers have added custom attributes whose ids conflict with the new ones on later platforms.&#xA;&#xA;Similarly, you can use tools:targetApi="11" in an XML file to indicate that the element will only be inflated in an adequate context.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<795208334>*